modules/data-loading/partials/load-part1-intro-and-schema.adoc

```diff
@@ -6,7 +6,7 @@ We will call out whether a particular step is common for all loading or specific
 == Example Schema
 This example uses part of the LDBC_SNB schema:
 
-[source,php]
+[source,gsql]
```

Comment:
Is there a gsql language type? or just sql?

Reply:
@wuqingjuntg  Yes, there is a gsql language type.